config/characterizeImage.py

```diff
@@ -7,24 +7,9 @@
 
 obsConfigDir = os.path.dirname(__file__)
 
-# Cosmic rays
-# These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
-config.repair.cosmicray.nCrPixelMax = 100000
-config.repair.cosmicray.cond3_fac2 = 0.4
```

Comment:
I'm not an expert on these configs, but I will note that following this ticket they will appear to change to:
```
# maximum number of contaminated pixels
config.repair.cosmicray.nCrPixelMax=10000

# used in condition 3 for CR; see CR.cc code
config.repair.cosmicray.cond3_fac2=0.6
```

It looks like [DM-40586](https://jira.lsstcorp.org/browse/DM-40586) has been set up to capture this change. Any conclusions on that ticket should apply back to DECam if appropriate.